queries.sql

```diff
@@ -15,3 +15,51 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = 'unspecified';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT mysavepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO my_savepoint;
```

Comment:
- [ ] The name of the SAVEPOINT you created is `mysavepoint` and in line 42, you do a ROLLBACK TO `my_savepoint.` I think there is some name discrepancy here. Kindly correct the name.

Reply:
Yes, sir .... my bad, when I copied the original command to the script file. I fixed it.

![FIX-02](https://github.com/jlcarrascof/Vet-clinic-database/assets/86327867/8e09bb93-3ba6-48a9-8af0-32f6248f3187)

And this is the prove that I ran my script well, when I was doing the exercise. This screenshot is above in the pull request's body.

![269510735-ac2bb854-33fe-4b17-a3a5-59d6d54f9f9e](https://github.com/jlcarrascof/Vet-clinic-database/assets/86327867/c5f245d0-7e9b-46e3-8b74-2ed91320c44a)

Thanks for your suggestions, @t-yanick dear reviewer.